### ISSUE TITLE
images for QC

### ISSUE THREAD

**Issue by jordandekraker** [khanlab/hippunfold#56]
Would be nice to have a few more slices for QC, perhaps something like this in the `vis_qc_dseg.py` script:

```
# original plot (centered)
fig = plotting.plot_roi(roi_img=snakemake.input.seg, bg_img=snakemake.input.img, display_mode='ortho',view_type='continuous',alpha=0.5,dim=-1,draw_cross=False)
fig.savefig('slice-2_' + snakemake.output.png)

# move 2mm backward in each direction
fig2 = plotting.plot_roi(roi_img=snakemake.input.seg, bg_img=snakemake.input.img, cut_coords=[x -2 for x in fig.cut_coords])
fig2.savefig('slice-1_' + snakemake.output.png)

# move 2mm forward in each direction
fig3 = plotting.plot_roi(roi_img=snakemake.input.seg, bg_img=snakemake.input.img,cut_coords=[x +2 for x in fig.cut_coords])
fig3.savefig('slice-3_' + snakemake.output.png)

# 3D surface
# requires `snakemake.input.surf`
fig4=plotting.plot_surf(snakemake.input.surf,view='dorsal')
fig4.savefig('3Dmodel' + snakemake.output.png)
```

**Comment by akhanf** [comment]
yes that would be great! I can start a PR unless you have a branch already?

one change I would suggest from what is here, is to add the files as outputs to the rule, rather than building the filenames in the script, so that snakemake is aware of them

**Comment by akhanf** [comment]
resolved in #57
